tests/mir-opt/inline/inline_coroutine.main.Inline.panic-abort.diff

```diff
@@ -16,7 +16,6 @@
 +     scope 3 (inlined Pin::<&mut {coroutine@$DIR/inline_coroutine.rs:20:5: 20:8}>::new) {
 +         debug pointer => _3;
```

Comment:
Shouldn't this one also be gone? Pin::new (after inlining the function call in its body and stripping debug info) is just the same as Pin::new_unchecked, but its debug info seems to stay around.

Reply:
But isn't the same true for the `pointer` arg of `new_unchecked`?

Ah, if the currently-being-optimized function were in libcore, the arg of `new` would also be gone...

Makes sense, but subtle